src/styles/images/index.md.njk

```diff
@@ -8,53 +8,144 @@ layout: layout-pane.njk
 
-### Photography
+This guidance is for government teams that build online services. To learn how to use images as a GOV.UK content publisher, go to [GOV.UK content guidance for images](https://www.gov.uk/guidance/content-design/images).
 
-The aspect ratio for photographs should be 3:2.
+## Using images to give information
 
-{{ example({group: "styles", item: "images", example: "default", html: false, open: false, size: "l"}) }}
+Services usually work best without relying on images. Focus on writing clear, simple content for your service first.
 
+Adding images can sometimes help users better understand instructions. However, images should only be visual aids — written content must contain all the information the user needs to complete the service.
 
-### Illustrations or representative imagery
+If you choose to add an image to your service, you’ll need to carefully decide if the image needs to be described with alternative text (alt text).
 
-If your image represents something physical, such as a letter, document or credit card you should use the aspect ratio of that item.
+## Photography
 
-{{ example({group: "styles", item: "images", example: "representative", html: false, open: false, size: "l"}) }}
+Use photography when it’s important to show a lifelike representation of something. For example, showing a photo of a document a user has uploaded.
 
+Do not use photography to represent abstract concepts, such as to convey an impression or emotion to the user. Avoid using generic stock photography in your service.
 
-## Alternative text
+## Illustrations
 
-This guidance is for government teams that build online services. [To learn about publishing alternative text in GOV.UK mainstream content, go to GOV.UK's 'Images' guidance.](https://www.gov.uk/guidance/content-design/images)
+Use illustrations to show a simplified image of something complex. For example:
+- to demonstrate a physical interaction, like scanning a passport
+- to help users find something on a document, like a reference number
+
+### Creating illustrations
+
+Use as few visual elements as possible to communicate your message.
+
+Start by using the [GOV.UK colour palette](/styles/colour/#colour-palette). If you need more colours, try to use tints of the colour palette.
+
+Use flat blocks of colour. Avoid using shadows or gradients, except when needed to show a distinctive feature of an object or to communicate layering or depth.
+
+If creating multiple illustrations for a service, use consistent colours, line widths and border radiuses across them all.
+
+Avoid depicting people in illustrations. If you need to show people, think carefully about how you represent the people that use your service — do not only show one kind of person. 
+For example, only depicting people of a particular age, gender or ethnicity can exclude people, reinforce negative stereotypes and cause harm.
```

Comment:
Using the words 'cause harm' is a bit out of place here. Lots of suggestions on this page are about not causing harm, like adding alt text.

Reply:
```suggestion
For example, only depicting people of a particular age, gender or ethnicity can exclude people and reinforce negative stereotypes.
```